kore/src/main/java/org/kframework/utils/StringUtil.java

```diff
@@ -433,38 +433,23 @@ private static String splitLine(String str, final int col) {
      * Finesse the JCommander usage output to make it more readable to the user.
      *
      * This function does two things. First, it reworks the indentation to fix a
```

Comment:
>This function does two things.

Only one thing now.

Reply:
I just removed the function and inlined it to the one use-case, now that basically only the first line was doing anything.